docs/cockpit.rst

```diff
@@ -23,29 +23,21 @@ the host you are running the ADB on.
 
 1. Start the ADB
 
-2. Log into the ADB
+2. Start cockpit
 
-   ``$ vagrant ssh``
+   ``$ vagrant service-manager restart cockpit``
```

Comment:
I am not in favor of another Vagrantfile for Cockpit, as it does does not fall in the categories of K8s, OpenShift, Mesos etc.